### DISCUSSION TITLE
Github Satellite India 2021

### DISCUSSION THREAD

**Discussion by manavmodi22** [EddieHubCommunity/support#1164] [Conference Notes]
### From the GitHub Universe 2021 please share what you learned or any tools you want to learn more about below 👇
![image](https://user-images.githubusercontent.com/55242552/112587645-fdeb7c00-8e23-11eb-8682-ca182c35b8d0.png)

**Comment by manavmodi22** [comment]
### GitHub Satellite Keynote
**The Key Announcements made at GitHub Satellite India 2021.** 🎉

- 𝐆𝐢𝐭𝐇𝐮𝐛 𝐆𝐫𝐚𝐧𝐭𝐬 for Open Source

- 𝐆𝐢𝐭𝐇𝐮𝐛 𝐀𝐜𝐜𝐞𝐥𝐞𝐫𝐚𝐭𝐞 for Startups 

- 𝐆𝐢𝐭𝐇𝐮𝐛 𝐄𝐱𝐭𝐞𝐫𝐧𝐬𝐡𝐢𝐩𝐬 for Students

Exciting programs for Developers in India!

![image](https://user-images.githubusercontent.com/55242552/112587440-946b6d80-8e23-11eb-9dca-f26d38197483.png)

**Comment by manavmodi22** [comment]
### GitHub Satellite Open Source Track: Talk 1
Narrating the story of how Hoppscotch scaled, it served as a true case study around the power of OpenSource. The product idea got over the night 100+ issues after the initial commit and today they have grown so much!

- Open Source has always been about the community. The more you engage, the more you grow. In the process, even people who contributed build onto the skill set they already have!

- Consistency is the key while working in Open Source.

- Monetizing Open Source?
1. Sponsorships and Donations through the Github Sponsor button
2. Self-hosted Instances
3. Educating users to increase interactions with your project.
4. Paid Features in the product that you are working on
5. Finding Investors
- Leveraging Open Source Developer Tools

1. Use of Open Source under the hood by using open-sourced tooling/solutions. 

- How to get started? 

1. Find a problem around you.
2. Ideate to solve the problem.
3. Build an MVP
4. Look for Enterprise Level. This isn't something complex. Enterprise is just made up of multiple single users. Scalability should be the end goal.
5. Engage with developer forums, subreddits, Discord communities with your product.

- Monetize early and scale upwards with time.
- **Document your progress.**-This is one of the key components to succeed in developing a Successful Open Source Project.
_**Question- Contributors hail from different timezones. How is it managed?**_

- The presenter actually used this as an advantage to their growth. The presenter and his friend managed the community and work in shifts. The different timezones helped their project through the constant process of contributions from developers around the world.

Reach out to @liyasthomas for any doubts.
Check out [Hoppscotch](https://github.com/hoppscotch/hoppscotch)

**Comment by manavmodi22** [comment]
### GitHub Satellite Open Source Track: Lightning Talk by @AmruthPillai
i18n-ize your app! Amruth Pillai made **_[rxresu.me](rxresu.me)_** which is a free open source resume builder. 

- Inclusivity is the key.
- The project tried to be inclusive by including languages around the world. Just a simple idea, and it opened an arena of opportunities for people around the world to contribute and add value.
### Adopting i18n 🚀
![Adopting i18n](https://user-images.githubusercontent.com/55242552/112637574-517dba00-8e64-11eb-8a1f-d601cb319dea.jpeg)

### Glossary around i18n 📕
![Glossary](https://user-images.githubusercontent.com/55242552/112637774-8427b280-8e64-11eb-8fef-f77368fb738e.jpeg)


Reach out to @AmruthPillai for any doubts.

**Comment by manavmodi22** [comment]
### GitHub Satellite Open Source Track: Talk on Helping fight COVID-19 using GitHub by @AnandChowdhary
The story of his achievements with the open-source project Karuna 2020 aimed at raising funds and distributing dry ration packs is truly what one must consider as the real power of @github 

- The site itself built on top of the open-source project [Emergency Site](https://github.com/maxboeck/emergency-site)
- The hack is built using Github actions to automate... What? Like almost everything.

1. The Donor receipt generation
2. The Recipient flow
3. Posting of Recipient Photo on the Website automation 
4. In order to protect the identity, the images blur automation

- Zero Server Cost using Github Pages. 
- Open Sourcing the project itself to help build other non-profits on top of it!
- They have published "Guides", which give insights into how all of this is achieved
GitHub Actions have emerged as a very powerful tool, and scheduling is just one of its pool of benefits.


Reach out to @AnandChowdhary for any doubts.

Here are some links given in GitHub Discussion for the talk:

Karuna 2020 website: [https://karuna2020.org](https://karuna2020.org)
Karuna 2020 on GitHub: https://github.com/Karuna2020
[How Clojure's concurrency model helped us feed 12000 people](https://krimlabs.com/blog/clojure-core-async-to-sustain-12k) by @shivekkhurana
[Karuna 2020: Using GitHub Actions in the COVID-19 response efforts](https://github.blog/2020-08-13-github-actions-karuna/) by @MishManners
COVID-19 topic on GitHub: [https://github.com/topics/covid-19](https://github.com/topics/covid-19)

**Comment by eddiejaoude** [comment]
Awesome ❤️  thank you for sharing!
